### PR TITLE
feat : add hanoi meetup

### PR DESCRIPTION
**Pull request by houssembaltii**
image url: https://unsplash.com/fr/photos/une-haute-tour-assise-a-cote-dune-foret-verdoyante-sju7BXYLhQc

### CODE REVIEW

File: src/content/meetup/2024-09-25-hanoi.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Fork it! Meetups Hanoi 2025
+name: Meetup Hanoi, Vietnam
+date: 2024-09-25
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-25-hanoi.mdx:4]: wrong date

File: src/content/meetup/2024-09-25-hanoi.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+title: Fork it! Meetups Hanoi 2025
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-25-hanoi.mdx:2]: 2024

File: src/content/meetup/2024-09-25-hanoi.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  address: Soon
+excerpt:
+      Fork It! is excited to host its debut meetup in Hanoi, Vietnam!
+      This event is designed for web development and UX enthusiasts to come together, 
+      share insights, and collaborate on new ideas. As our community expands, we’re bringing this unique experience
+      to Vietnam for the first time, providing a platform for learning, networking, and engaging discussions.
+      Follow us for updates and be ready to sign up soon to join this dynamic event . 
+
+image:
+  src: "/meetups/2024-09-25-hanoi.jpg"
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-25-hanoi.mdx:16]: Same wrong date, you also have to change the name of the file

File: src/content/meetup/2024-09-25-hanoi/about.tsx [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                    />
+                    <span>
+                      <time
+                        dateTime={props.meetup.date}
+                        className="font-semibold text-primary"
+                      >
+                        {formatDateTime(props.meetup.date)}.
+                      </time>{" "}
+                      Come enjoy conferences and a networking cocktail to wrap
+                      up September on a high note.
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-25-hanoi/about.tsx:65]: It's not on September

File: src/content/meetup/2024-09-25-hanoi/about.tsx [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+                    </span>
+                  </li>
+                )}
+              </ul>
+              <p>Description for the location&apos;s meetup</p>
```
Comment (JeanneGrenet) [inline src/content/meetup/2024-09-25-hanoi/about.tsx:70]: Same comment than KL, you can't put that on the website I think, it should be a comment

**Review by JeanneGrenet — CHANGES_REQUESTED** [review]

**Comment by houssembaltii** [comment]
sorry for the incorrect dates, at first , i only have the city where the event will be held in .

**Review by JeanneGrenet — APPROVED** [review]

**Pull request merged**